### PR TITLE
HTML built, header stylized

### PR DESCRIPTION
**Pull request by Sergei-Nikolaienko**
https://Sergei-Nikolaienko.github.io/layout_miami/

### CODE REVIEW

**Review by liliia-livandovskaya — CHANGES_REQUESTED** [review]
Deploy project Screenshot 2022-12-09 at 11 40 15

**Review by OlEzhA0 — CHANGES_REQUESTED** [review]
привіт
записав тобі відео фідбек по твоїй роботі
[link](https://drive.google.com/file/d/1ujwPBwdKxzxI8yiXY1DmAnK69mCdkAST/view?usp=sharing)
якшо немає доступу - пінгуй мене в fe_chat

**Comment by Sergei-Nikolaienko** [comment]
> 

Дякую за фідбек - в тому коміті зробив тільки хедер (в завданні казали кожен блок на перевірку кидати), зараз запушу все інше

**Comment by witflash** [comment]
Hey! You lost some requirements:
> 8. Pictures in Gallery should increase on hover
> 9. Location-related addresses / links should open google maps in a new tab target="_blank"
> 10. Apply :hover effect for images on page (testimonials / gallery, other sections).
> 17. Fix menu for small screens (if there is not enough space for all the menu items)

File: src/scripts/main.js [hunk 1]
```diff
@@ -1 +1,21 @@
...
+
+window.addEventListener('hashchange', () => {
+  if (window.location.hash === '#menu') {
+    document.body.classList.add('page__body--with-menu');
+  } else {
+    document.body.classList.remove('page__body--with-menu');
+  }
+});
+
+const toTopBtn = document.getElementById('topButton');
```
Comment (witflash) [inline src/scripts/main.js:11]: If you want to use some elements in JS, you better add `js-{some-name}` class and use it in your script.
Here is better do smth like this:
```
const toTopBtn = document.querySelector('.js-top-button');
```

File: src/styles/blocks/card.scss [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    font-size: 24px;
+    line-height: 140%;
+  }
+
+  @include onIpad {
+    width: 392px;
+  }
+  @include onDesktop {
+    width: 324px;
+  }
```
Comment (witflash) [inline src/styles/blocks/card.scss:35]:
```suggestion
  @include onIpad {
    width: 392px;
  }

  @include onDesktop {
    width: 324px;
  }
```

File: src/styles/blocks/compare.scss [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+    &__wrapper {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+    }
+
+    &__card {
+      margin-bottom: 48px;
+      &:last-child {
```
Comment (witflash) [inline src/styles/blocks/compare.scss:26]:
```suggestion
      margin-bottom: 48px;
      
      &:last-child {
```

File: src/styles/blocks/features.scss [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+      padding-bottom: 100%;
+    }
+
+    @include onIpad {
+      height: 186px;
+      padding-bottom: 0;
+
+      &::after {
+        content: "";
+        display: block;
```
Comment (witflash) [inline src/styles/blocks/features.scss:41]: You don't need to duplicate `content` and `display` rules in media queries if they did not change.

File: src/styles/utils/_vars.scss [hunk 5]
```diff
@@ -1 +1,7 @@
-$c-gray: #eee;
+$ipad-size: 600px;
+$desktop-size: 1024px;
+
+:root {
+  --primary-grey: #1d1d1d;
+  --form-background: #f8f8fa;
```
Comment (witflash) [inline src/styles/utils/_vars.scss:6]: It's better to name it:
```
--color-primary
--color-background or --color-secondary
```

File: src/styles/blocks/title.scss [hunk 6]
```diff
@@ -0,0 +1,15 @@
+.title {
+  font-weight: 500;
+  font-size: 32px;
+  letter-spacing: -1.5px;
```
Comment (witflash) [inline src/styles/blocks/title.scss:4]: Do not use non-integer values. Use here `1` or `2` px.

File: src/styles/blocks/page.scss [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+      transform: translateX(0);
+    }
+  }
+
+  &__body {
+    cursor: url("../images/Cursor-Mouse-Mice.svg"), auto;
+    background: var(--primary-grey);
+    color: white;
+
+    a {
```
Comment (witflash) [inline src/styles/blocks/page.scss:55]: Do not use CSS rules for tags. Use it for necessary blocks and elements.

File: src/styles/blocks/menu.scss [hunk 8]
```diff
@@ -0,0 +1,110 @@
+.menu {
+  background: #e5e5e5;
+  text-align: left;
+  z-index: 1000;
```
Comment (witflash) [inline src/styles/blocks/menu.scss:4]: Do you really need such a big z-index? If you do not have a large project, you can use smaller values (e. g. from 1 to 10)

File: src/styles/blocks/header.scss [hunk 9]
```diff
@@ -0,0 +1,101 @@
...
+
+  &__top-panel {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__logo {
+
+    &-image {
```
Comment (witflash) [inline src/styles/blocks/header.scss:38]: Don't divide elements' names. Use full name: 
```
&__logo-image {
```

**Review by witflash — CHANGES_REQUESTED** [review]
Hey! 
Pay attention to my comments.

And you lost some requirements:
> 8. Pictures in Gallery should increase on hover
> 9. Location-related addresses / links should open google maps in a new tab target="_blank"
> 10. Apply :hover effect for images on page (testimonials / gallery, other sections).
> 17. Fix menu for small screens (if there is not enough space for all the menu items)

**Review by witflash — CHANGES_REQUESTED** [review]
Привіт!
А по цим вимогам із завдання я не бачу змін:

> And you lost some requirements:
> > 8. Pictures in Gallery should increase on hover
> > 9. Location-related addresses / links should open google maps in a new tab target="_blank"
> > 10. Apply :hover effect for images on page (testimonials / gallery, other sections).
> > 17. Fix menu for small screens (if there is not enough space for all the menu items)

Якщо є якісь питання, чи щось не зрозуміло, можеш звернутись в канал fe_chat

**Comment by ivanchuk-yurii** [comment]
should be in one line
image
in one line 
image
fix hover
https://user-images.githubusercontent.com/53611910/208253393-877beadd-9255-4791-92da-6ffb23831d7a.mov
make all input in form required. don't reload page on form submit

**Review by liliia-livandovskaya — CHANGES_REQUESTED** [review]
Great job! Your landing looks awesome 👍 
There are two more things to improve:

1. Add `transition-duration` on hover

2. Pay attention to the title and fix it according to the mockup 
Screenshot 2022-12-19 at 13 52 58

**Review by witflash — APPROVED** [review]
Pretty well!
Well done! 👍

**Pull request closed without merging**